csb/src/main/scala/edu/msstate/dasi/csb/EdgeData.scala

```diff
@@ -88,8 +87,7 @@ object EdgeData {
             origPkts.toLong,
             origIpBytes.toLong,
             respPkts.toLong,
-            respIpBytes.toLong,
-            desc
+            respIpBytes.toLong
           )
         // TODO: check why we need the following, i.e. why might "desc" be empty?
         case Array(ts, origPort, respPort, proto, duration, origBytes, respBytes, connState, origPkts, origIpBytes,
```

Comment:
Since we don't have the `desc` field anymore, this second `case` block can be safely deleted. Also, I would handle again any int/long/double conversion inside a separate try/catch block as in `DataParser::logToGraph()`